lib/cli/test/unit/Cardano/CLISpec.hs

```diff
@@ -115,6 +117,7 @@ spec = do
             , "  wallet                   Manage wallets."
             , "  transaction              Manage transactions."
             , "  address                  Manage addresses."
+            , "  stake-pool               Interact with stake pools."
```

Comment:
Hmm... why not just "Manage stake pools."?

Reply:
fixed